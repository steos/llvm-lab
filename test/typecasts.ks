
	void main() {
		float x = 9.9f;
		double y = 9e99;
		double z = 3.1415;
		
		byte bar = (byte)1024;
		
		long foo = 3;
		
		iprintln((int)bar);
		iprintln((int)x);
		iprintln((int)y);
		iprintln((int)z);
	}
	
	native void iprintln(int);