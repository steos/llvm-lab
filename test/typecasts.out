0
9
-2147483648
3
